### DISCUSSION TITLE
local exporter as <file_name>.proto file

### DISCUSSION THREAD

**Discussion by skmonjurul** [open-telemetry/opentelemetry-collector#2463] [Q&A] [answered]
Is .proto file supported by exporter?

receivers:
  otlp:
    protocols:
      grpc:

exporters:
  file:
    path: ./sampleProto.proto

service:
  extensions: []
  pipelines:
    traces:
      receivers: [otlp]
      exporters: [file]

Is this valid? Because I am not getting any error and sampleProto.proto file is getting created but the data is in json format.

**Comment by bogdandrutu** [comment, marked as answer]
By ".proto" you mean binary proto, not the proto definition :)

See https://github.com/open-telemetry/opentelemetry-collector/discussions/2457 where we discussed exactly this.
